lib/v2/i18n/profile_screens/profile/profile.i18n.dart

```diff
@@ -9,7 +9,7 @@ extension Localization on String {
           "Upgrading account ...": 'Actualizando cuenta ...',
           'Visitor': 'Visitante',
           'Resident': 'Residente',
-          'to': 'a',
+          'to': 'A',
```

Comment:
This localization needs to use a pattern. 

// Prints 'Hello John, this is Mary' in English.
// Prints 'Olá John, aqui é Mary' in Portuguese.
print("Hello %s, this is %s".i18n.fill(["John", "Mary"]));

See here -> https://pub.dev/packages/i18n_extension

Reply:
Hi Nick, I see your point and agree, I need to change some of them, however I dont think this apply to this case. 
Please see screenshot. 
![image](https://user-images.githubusercontent.com/51983923/130258566-42fffba8-8945-4ae7-88af-ccf8649f7759.png)